src/components/YouTube.tsx

```diff
@@ -15,7 +15,14 @@ const Figure = styled.figure`
  * e.g. For https://www.youtube.com/watch?v=H-O3r2YMWJ4&t=123 the `start` is 123 (which means 123 seconds)
  * @returns Embedded YouTube video component
  */
-const YouTube = ({ id, start, title }) => {
+
+export interface IProps {
+  id: string
+  start: string
```

Comment:
The start prop should be optional here.

Reply:
Is this correct?
```start?: string```

Reply:
@motemotech I just pushed a commit to make `start` and `title` as optional, and I gave them a default value.

Reply:
Thank you so much!